core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-scheduling/dot-experiments-configuration-scheduling.component.html

```diff
@@ -66,4 +66,3 @@ <h2 class="flex align-items-center gap-1 uppercase" data-testId="scheduling-card
 </ng-container>
 
 <ng-container dotDynamic></ng-container>
-<p-confirmPopup></p-confirmPopup>
```

Comment:
No needed it?

Reply:
No. we usually add one to the shell and the service finds the component automatically. But sometimes we added one in the child component because of the test.